schema/check_schemas.py

```diff
@@ -60,37 +60,50 @@ def save_schema_validation_summary(self, validation_status):
         return output_filename
 
 
-def parallel_validate_schema(validator, file_names):
-    num_processors = multiprocessing.cpu_count()
-    logging.info('Schema validation: %s processors for %s schema validations', num_processors, len(file_names))
+def validate_all_schema(validator, file_names):
+    if not validator.options.run_serial:
```

Comment:
flip the "then" and "else" branches so that you can avoid using `not` in your condition test.